demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -180,6 +183,9 @@ <h5 class="modal-title" id="additional-options-modal-label">Additional Options</
             <option value="default">Meeting Default</option>
             <option value="vp8">VP8</option>
             <option value="h264ConstrainedBaselineProfile">H.264 Constrained Baseline Profile</option>
+            <option value="h264BaselineProfile">H.264 Baseline Profile</option>
+            <option value="h264MainProfile">H.264 Main Profile</option>
+            <option value="h264HighProfile">H.264 High Profile</option>
```

Comment:
So I guess we don't bother putting constrained high in as a demo option

Reply:
Actually, I'm looking at an iOS safari SDP now and it seems to include only constrained high, and not high. I think i'll look more into the fallback logic we were discussing.

Reply:
Made change on backend to forward CH as H, and included Constrain High when 'High' option is chosen.